tools/storybook/webpack.config.js

```diff
@@ -40,8 +40,14 @@ module.exports = ( storybookConfig ) => {
 		'./setting.mock.js'
 	);
 
-	storybookConfig.resolve.alias[ 'react/jsx-runtime' ] =
-		require.resolve( 'react/jsx-runtime' );
+	storybookConfig.resolve.alias.react = path.resolve(
+		__dirname,
+		'./node_modules/react'
+	);
+	storybookConfig.resolve.alias[ 'react-dom' ] = path.resolve(
+		__dirname,
+		'./node_modules/react-dom'
+	);
 
 	// We need to use react 18 for the storybook since some dependencies are not compatible with react 17
```

Comment:
Could we also take care of this comment or create a separate ticket for it? Storybook uses React 18 since https://github.com/woocommerce/woocommerce/pull/51037

Reply:
This comment explains why the code below is necessary:

![image](https://github.com/user-attachments/assets/23b3fa54-18e1-4ddd-a985-627bcf956749)